requirements/edx/post.txt

```diff
@@ -9,4 +9,9 @@
 # trying to compile lxml as a dependency causes setuptools to go into an
```

Comment:
Maybe we can reword this content to something along the lines of:

"python-saml depends on lxml, which is referenced in base.txt.  A bug exists in setuptools (version?) which results in an infinite loop during resolution of this dependency during compilation.  So we need to install python-saml only *after* lxml has been successfully installed.  In addition, we are currently utilizing a forked version of python-saml, managed by OpenCraft, which features enhanced logging.  We will return to the official version of python-saml when https://github.com/onelogin/python-saml/pull/159 (or its derivative) has been incorporated into the main project."

"